server/dba/seeders/01-christmasTreesForests.js

```diff
@@ -1,8 +1,8 @@
-'use strict';
+
```

Comment:
Formatting and set forest end dates to the future so dev and tests have expected data. A future PR will address this more generally for the test environment to NOT be dependent on the seeds file.